### PR TITLE
Measurements checks page

### PR DESCRIPTION
**Pull request by fsoubelet**
Including content from our discussion and Tobias' presentation on Monday.

Please proof-read!

### CODE REVIEW

**Comment by fsoubelet** [comment]
For reference the updated slides are at https://indico.cern.ch/event/1140179/contributions/4784376/attachments/2416087/4139200/omc_march2022.pdf

I'm not 100% sure on the local corrections sign-swapping thingy, I think I remember there are cases of double-switching but not for which magnets (local tilted? idk)

**Comment by fsoubelet** [comment]
Reminder: one can check the build by going to the `Checks` tab in this PR, then click on the second build workflow (the one that has `on: pull_request`) and download the `site-build` artifact.

For the last commit the link is: https://github.com/pylhc/pylhc.github.io/suites/5896797580/artifacts/200473167
Then, unzip the downloaded archive and open `index.html`.

File: docs/procedures/general_checks.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# General Checks
+
+## Before Measuring
+
+Here are a few checks to perform before starting measurements.
+These are updated as much as possible for Run 3 values.
+
+- [ ] <details class="nodeco"><summary>Make Sure Intensity Is below $10^{10}$ppb until Local Corrections Are In</summary>
+      <p> After local corrections, we can move to having 3 bunches at $10^{10}$ppb evenly spaced along the ring.
```
Comment (tpersson) [inline docs/procedures/general_checks.md:9]: Every single bunch still needs to be below 10^10

File: docs/procedures/general_checks.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+- [ ] <details class="nodeco"><summary>Turn Off the Landau Octupoles</summary>
+      <p> Unless you specifically need them on for your measurements (ampdet & co).
+      These are the MO elements.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Correct the Machine Chromaticity to a Few Units</summary>
+      <p> In the past we've used $2$, which we will most likely keep using in Run 3.
+      </p></details>
+
+- [ ] <details class="nodeco"><summary>Mask the ATLAS BCM</summary>
```
Comment (tpersson) [inline docs/procedures/general_checks.md:25]: We cannot change this setting from the CCC. The only way is through contacting the ATLAS control room. Talk to the EIC. At least in the beginning we should double check that the multiturn is actually giving the right indication.

File: docs/procedures/general_checks.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      - [ ] <details class="nodeco"><summary>Calculated Global Corrections</summary>
+            <p> The calculated global correction are really corrections and should be trimmed in with a positive sign.
+            </p></details>
+      - [ ] <details class="nodeco"><summary>Skew Magnets</summary>
+            <p> All skew magnets are inversed in LSA / MAD-X conventions and the calculated corrections for these should be trimmed in with an opposite sign (that means calculated correction * -1).
+            </p></details>
+      - [ ] <details class="nodeco"><summary>Triplets</summary>
+            <p> The triplets are tricky as all three are powered in series with a common knob, then some have an additional trim.
+            Calculated corrections should be *sign-swapped for Q1 and Q3 but not for Q2*.
+            See with the EIC if you have any doubts.
```
Comment (tpersson) [inline docs/procedures/general_checks.md:93]: I don't we can rely on them remembering this convention. This is very specific, what they can help with is creating a knob and also the permissions etc.

File: docs/procedures/general_checks.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            </p></details>
+      - [ ] <details class="nodeco"><summary>Local Corrections</summary>
+            <p> Decide on a convention for how we output the corrections (omc3 or bbsrc) and how to use them? Do checks on friday for that!
+            </p></details>
+
+- [ ] <details class="nodeco"><summary>Knob Creation Permission</summary>
+      <p> It is not enough to be logged in as `lhcop` to create knobs for corrections.
+      Both Tobias and Ewen have the permissions, if they aren't present see with the EIC.
+      </p></details>
```
Comment (tpersson) [inline docs/procedures/general_checks.md:102]: They might not have the permission but they can give themself or anyone in the team the permission

**Review by tpersson — APPROVED** [review]
Looks good! I made some comments where I thought some clarifications could help.  I don't think I need to review again so just clarify and then it is fine to merge from my side.

File: docs/procedures/general_checks.md (hunk 1 above)
Comment (fsoubelet) [reply docs/procedures/general_checks.md]: Done in [a5c6908](https://github.com/pylhc/pylhc.github.io/pull/69/commits/a5c6908372ef4055d5b99ee516d3330401ec7eae)

File: docs/procedures/general_checks.md (hunk 2 above)
Comment (fsoubelet) [reply docs/procedures/general_checks.md:25]: Done in [a5c6908](https://github.com/pylhc/pylhc.github.io/pull/69/commits/a5c6908372ef4055d5b99ee516d3330401ec7eae)

File: docs/procedures/general_checks.md (hunk 3 above)
Comment (fsoubelet) [reply docs/procedures/general_checks.md]: Done in [a5c6908](https://github.com/pylhc/pylhc.github.io/pull/69/commits/a5c6908372ef4055d5b99ee516d3330401ec7eae)

File: docs/procedures/general_checks.md (hunk 4 above)
Comment (fsoubelet) [reply docs/procedures/general_checks.md:106]: Done in [a5c6908](https://github.com/pylhc/pylhc.github.io/pull/69/commits/a5c6908372ef4055d5b99ee516d3330401ec7eae)

**Pull request merged**